app/graph/Chat.tsx

```
@@ -311,15 +329,19 @@ export default function Chat({ onClose }: Props) {
                             break;
 
                         case "error":
-                            const statusCode = Number(line.split("status:")[1].split(" ")[0]);
+                            const statusCode = getStreamStatusCode(line);
```

Biome (2.4.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)